_pages/sample-proj.md

```diff
@@ -1,56 +0,0 @@
----
-layout: page
```

Comment:
i'd have preferred this to be taken out in another PR :P but oh well

Reply:
oh well :p my bad